UI/window-basic-main.cpp

```diff
@@ -3089,10 +3089,29 @@ void OBSBasic::UpdateContextBarDeferred(bool force)
 				  Qt::QueuedConnection, Q_ARG(bool, force));
 }
 
-void OBSBasic::SourceToolBarActionsSetEnabled(bool enable)
+void OBSBasic::SourceToolBarActionsSetEnabled()
 {
+	bool enable = false;
+	bool disableProps = false;
+
+	OBSSceneItem item = GetCurrentSceneItem();
+
+	if (item) {
+		OBSSource source = obs_sceneitem_get_source(item);
+		const char *id = obs_source_get_id(source);
+
+		if (strcmp(id, "scene") == 0 || strcmp(id, "group") == 0)
+			disableProps = true;
+
+		enable = true;
+	}
+
+	if (disableProps)
```

Comment:
You can use `obs_source_configurable` for this instead of checking the source id.

Reply:
Fixed